pages/interface/components/Confetti/index.js

```diff
@@ -1,15 +1,7 @@
 import ReactConfetti from 'react-confetti';
 import { useEffect, useState } from 'react';
 
-export default function Confetti({
-  width = 0,
-  height = 0,
-  recycle = false,
-  numberOfPieces = 800,
-  tweenDuration = 15000,
-  gravity = 0.15,
-  ...props
-}) {
+export default function Confetti({ width = 0, height = 0, ...props }) {
```

Comment:
```suggestion
export default function Confetti({
  width = 0,
  height = 0,
  recycle = false,
  numberOfPieces = 800,
  tweenDuration = 15000,
  gravity = 0.15,
  ...props
}) {
```

Ao invés de excluir, repasse essas propriedades para o componente `ReactConfetti` mais abaixo